README.md

```diff
@@ -3,18 +3,104 @@
 This application allows users to create and manage work orders.
 
 # Setup
-## Dev environment
+## Database
 To create the databases for the dev environment: `rake db:setup`
 
+## Broker
+To create the exchanges, queues and usernames etc. use the GitLab repo: [aker-environments](https://gitlab.internal.sanger.ac.uk/aker/aker-environments)
+
 # Testing
 To run ALL the tests for this project: `bundle exec rake`
 
 To run just the rspec tests: `bundle exec rspec`
 
-Note that if any changes are made in the `app/client`,`app/assets` or `app/javascript` directories, any files in the `public/assets`,`public/packs` or `public/packs-test`  directory must be removed, and webpack must be run with the `webpack.production.js` configuration file:
+# Misc.
+## Assets
```

Comment:
Don't we only have to do this for set shaper? 😨

Reply:
Hmmm... I think now that it uses React we need to be aware of assets.